lib/cryptography/sign/default.ex

```diff
@@ -19,6 +19,13 @@ defmodule Kadena.Cryptography.Sign.Default do
   @edwards_curve_ed :ed25519
 
   @impl true
+  def sign(msg, nil) do
+    with bin_msg <- blake2b_hash(msg, byte_size: 32),
+         hash <- url_encode64(bin_msg) do
+      {:ok, SignCommand.new(hash: hash)}
```

Comment:
@Odraxs this function is not really clear to me, no matter what it will return an error tuple.

![image](https://user-images.githubusercontent.com/1649973/206732442-cfe5ea35-b9e6-49c7-be6e-2488f2ff0bbf.png)

Are you really signing a message? (The answer to this question will define the responsibility of your function)

Reply:
we modify the `SignatureWithHash` type, which is called in the `new/1` function of the `SignCommand`, so that it receives a sign nil and we can obtain the hash.
![image](https://user-images.githubusercontent.com/80716239/206741572-b18b2014-bfef-4658-905d-fbb5cfae0312.png)

Reply:
@EdwinGuayacan ok, I understood you but IMO, it is not a signature, it is just a hash and a command without a signature. 

Why process it through the `sign/2` function and call it a signature?